Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.ps1

```diff
@@ -1841,6 +1864,28 @@ function CaseHoldRuleDeleteCommand([SecurityAndComplianceClient]$client, [hashta
     return $human_readable, $entry_context, $raw_response
 }
 
+function CaseHoldPolicySetCommand([SecurityAndComplianceClient]$client, [hashtable]$kwargs){
+    $enabled = ConvertTo-Boolean $kwargs.enabled
+    $add_exchange_locations = ArgToList $kwargs.add_exchange_locations
+    $add_sharepoint_locations = ArgToList $kwargs.add_sharepoint_locations
+    $add_public_locations = ArgToList $kwargs.add_public_locations
+    $remove_exchange_locations = ArgToList $kwargs.remove_exchange_locations
+    $remove_sharepoint_locations = ArgToList $kwargs.remove_sharepoint_locations
+    $remove_public_locations = ArgToList $kwargs.remove_public_locations
```

Comment:
The arguments names must match their names in the `yml` file.

Reply:
@BEAdi , if we change these argument names we should also change them in the `o365-sc-set-search` command and its corresponding `SetSearch` function. I originally aligned the argument names to this command.

Reply:
Changed both `o365-sc-set-search` and `o365-sc-case-hold-policy-set` with new argument names.